imsim/photon_ops.py

```diff
@@ -423,25 +440,20 @@ class XyToV:
     vector of the photons before entering the telescope.
 
     The transform is composed of the chain
-    (x,y) -> (u,v) -> (ra,dec) -> (thx, thy) -> v.
+    (x,y) -> (ra,dec) -> (thx, thy) -> v.
 
     The transform takes 2 vectors of shape (n,) and returns
     a column major vector of shape (n,3).
     """
 
-    def __init__(self, local_wcs, icrf_to_field, sky_pos):
+    def __init__(self, local_wcs, icrf_to_field, img_wcs):
```

Comment:
I think this doesn't need local_wcs anymore.  If we're breaking the API (with the change in the last parameter), might as well clean it up and remove local_wcs as well.

Reply:
Done. This also leaves some of the `photon_velocity` methods in `RubinOptics`, `RubinDiffraction` and `RubinDiffractionOptics` with unnecessary `local_wcs` and, now I look at them, also `rng`. Would you prefer those left alone or removed as well?

Reply:
Might as well clean up all the obsolete API.  If any of them seem like user-facing code, we should make this a deprecation rather than completely break old code.  But I think most of these functions are internal, not user facing, so it should be ok.

Reply:
I forgot to mention this in the meeting! Yes, everything looks to be internal to the photon ops themselves. `rng` can't be removed though, as it turns out, or it breaks RubinDiffractionOptics. Otherwise, tidied up as requested.